src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -240,7 +240,9 @@ var DOMPropertyOperations = {
         var propName = propertyInfo.propertyName;
         if (propertyInfo.hasBooleanValue) {
           // No HAS_SIDE_EFFECTS logic here, only `value` has it and is string.
-          node[propName] = false;
+          if (node[propName] !== false) {
```

Comment:
This is reading directly from the node (during a write phase?).  Is this going to cause dom thrash?  @spicyj @syranide